pkg/apis/bugs/v1/types.go

```diff
@@ -0,0 +1,22 @@
+package v1
+
+import "time"
+
+// Bug is used to represent bugs in some serialized content.  It also appears to be used internally for data tracking
+type Bug struct {
+	BugzillaBug  `json:",inline"`
+	Url          string `json:"url"`
+	FailureCount int    `json:"failureCount,omitempty"`
+	FlakeCount   int    `json:"flakeCount,omitempty"`
+}
+
+// BugzillaBug matches the bugzilla API.  We cannot change this and should consider writing a converter instead of having
+// a serialization we don't own.
+type BugzillaBug struct {
+	ID             int64     `json:"id"`
+	Status         string    `json:"status"`
+	LastChangeTime time.Time `json:"last_change_time"`
+	Summary        string    `json:"summary"`
+	TargetRelease  []string  `json:"target_release"`
+	Component      []string  `json:"component"`
```

Comment:
nit: `Components` ?

Reply:
this is driven by the ci-seach api:
https://search.ci.openshift.org/v2/search?search=storage&type=bug

```
bugInfo": {

    "id": 1867757,
    "status": "VERIFIED",
    "resolution": "",
    "severity": "medium",
    "priority": "unspecified",
    "summary": "Rebase node-registrar sidebar with latest version",
    "keywords": [ ],
    "whiteboard": "",
    "cf_internal_whiteboard": "",
    "creator": "hekumar@redhat.com",
    "component": [
        "Storage"
    ],
    "assigned_to": "hekumar@redhat.com",
    "creation_time": "2020-08-10T17:07:59Z",
    "last_change_time": "2020-08-25T01:42:05Z",
    "cf_environment": "",
    "target_release": [
        "4.6.0"
    ]

}
```

Reply:
(which in turn is a more or less direct mapping of the BZ api)